client/account-recovery/components/account-recovery-error-message/index.jsx

```diff
@@ -16,6 +16,12 @@ class AccountRecoveryErrorMessage extends Component {
 			case 'RestInvalidKeyError':
 				return translate( "We've failed to validate with the given code. " +
 					'Please double check if the code is correct.' );
+
+			// The error messages from our backend are quite descriptive in this case. e.g.
+			// "You've recently used this password", "Password must be at least 6 characeters.", etc.
+			// Thus I'm using it directly here.
+			case 'RestBadPasswordError':
+				return error.message;
```

Comment:
From what I'm seeing in the backend, they are all translated. However, I'm not sure if the API would respond according to the locale. I presumed it when I wrote this, but I think we need to confirm that. Do you know how can I test this?